tidy3d/components/simulation.py

```diff
@@ -210,6 +211,13 @@ class AbstractYeeGridSimulation(AbstractSimulation, ABC):
         "``autograd`` gradient processing.",
     )
 
+    post_norm: Union[float, FreqDataArray] = pydantic.Field(
+        1.0,
+        title="Post Normalization Values",
+        description="Factor to multiply the fields by after running, "
+        "given the adjoint source pipeline used. Note: this is used internally only.",
+    )
```

Comment:
Can we clarify this a little? I don't understand what `given the adjoint source pipeline used` means. Also, should we add the note about internal only to the `simulation_type` field too?

Reply:
Yea maybe more clear is "Factor to multiply the fields by to incorporate the adjoint source amplitudes.".

And yea we can add this note to simulation type too